source/php.txt

```diff
@@ -72,6 +72,44 @@ tutorials.
 
   - `Extension <https://github.com/mongodb/mongo-php-driver>`__
 
+.. _php-driver-arch:
+
+Driver Architecture
+-------------------
+
+This section describes how the components of the PHP driver work together.
+These components fit into the following general categories:
+
+- High-Level API, which includes the library and other integrations 
+- Extension, which includes the extension that integrates the system libraries
+- System, which includes the C Driver, BSON library, and encryption library
+
+The following diagram illustrates the architecture of the PHP driver
+components:
+
+.. figure:: /figures/PHP_driver_architecture.svg
+   :alt: PHP driver component architecture
+   :figwidth: 600px
+   
+The PHP library provides an API that is consistent with the other
+drivers, and it is continually updated to meet cross-driver
+specifications. You must add the library as a dependency in most
+applications that use PHP to work with MongoDB.
```

Comment:
The phrasing on this is a bit confusing to me. One question and one suggestion:

Q: This states that it's needed in "most applications" - does that mean that there are some instances where you don't need to the library in order to work with MongoDB in a PHP application? Or can we get rid of "most"?

S: If I'm understanding what's being said here, I think this could be a little more clear
```suggestion
specifications. You must add the library as a dependency to work with MongoDB in most
PHP applications.
```

Reply:
I think that there might be cases where a user could use only the extension to perform operations in their application, but this is not recommended.